tutorials/4 Plugin Developer/Create new QRenderer.ipynb

```diff
@@ -276,65 +307,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# The entire QDesign uses three tables. \n",
     "a_skeleton.write_qgeometry_table_names_to_file('./simple_output.txt')"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# The RouteMeander QComponent does not populate the junction or poly table, so the output of this should show only the path table.\n",
-    "# This usage of the method is requesting the output of just one component within QDesign. \n",
-    "a_skeleton.write_qgeometry_table_names_to_file('./simple_output_cpw1.txt',highlight_qcomponents=['cpw1'])"
+    "Here another example where we subselect a single QComponent instance (`cpw1`) of type RouteMeander.  This will only export the name of tables containing shapes related to that instance, which in this case is only paths, and not junctions or poly."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "Reminder: You can look at QGeometry tables by name of table, or name of by QComponent.\n",
```

Comment:
Please put the back in.

Reply:
I do not think this belongs in a Create QRenderer notebook. If needed we should find space for this in some other notebook. Let's go over the notebook together some other time. For now it was the cleanup pass, another time it will be the add pass

Reply:
The fundamental thing a QRenderer does is uses the QGeometry tables to "do something". The plug-in developer needs to know how to do this.  This is not clean up.